malduck/procmem/binmem.py

```diff
@@ -49,6 +52,13 @@ def image(self: T) -> Optional[T]:
                 self._image = self.__class__.from_memory(self, image=True)
             return self._image
         except Exception:
+            import traceback
+
+            log.debug(
+                "image construction throwed exception: %s",
+                traceback.format_exc(),
+                exc_info=True,
+            )
```

Comment:
It doesn't have sense to use both `traceback.format_exc` and `exc_info`
```suggestion
            log.debug(
                "image construction throwed exception",
                exc_info=True,
            )
```